debian/server-connect/Dockerfile

```diff
@@ -39,3 +39,6 @@ RUN echo "===> Installing JDBC, Elasticsearch and Hadoop connectors ..." \
 
 RUN echo "===> Installing GCS Sink Connector ..."
 RUN confluent-hub install confluentinc/kafka-connect-gcs:latest --no-prompt
+
+ENV CONNECT_PLUGIN_PATH=/usr/share/java/,/usr/share/confluent-hub-components/
```

Comment:
Actually, should this be set in `debian/server-connect-base` instead of here? That way images based on that image (e.g., `cp-enterprise-replicator` and `cp-enterprise-replicator-executable`) will have this property set correctly.

Reply:
Done.